test/regress/regress0/fp/issue-5524.smt2

```diff
@@ -2,5 +2,5 @@
 ; EXPECT: unsat
 (set-logic QF_FP)
 (declare-fun fpv5 () Float32)
-(assert (fp.eq (fp.mul RTP fpv5 fpv5) ((_ to_fp 8 24) RTN 0.04)))
+(assert (fp.eq (fp.mul RTP fpv5 fpv5) ((_ to_fp 8 24) #x3d23d70a)))
```

Comment:
Is this change necessary? I'd rather disable the dump-tester for this benchmark instead of changing this.  Or I thought you mentioned that changing the logic to e.g. `QF_FPLRA` ?

Reply:
I thought we didn't want to modify the logic. Reverted the changes and added `LRA` to the 2 FP regressions.

Reply:
Ok, yes, I think changing the logic to make it SMT-LIB compliant is the better fix here.